gp-includes/things/translation.php

```diff
@@ -628,13 +628,50 @@ public function set_status( $status ) {
 			$updated = $this->save( array( 'user_id_last_modified' => get_current_user_id(), 'status' => $status ) );
 		}
 
+		$this->save_feedback();
+
 		if ( $updated ) {
 			gp_clean_translation_set_cache( $this->translation_set_id );
 		}
 
 		return $updated;
 	}
 
+	public function save_feedback() {
+		global $wpdb;
+		if ( ! GP::$permission->current_user_can( 'approve', 'translation', $entry->id, array( 'translation' => $entry ) ) ) {
+			return false;
+		}
+		if ( GP::$permission->current_user_can( 'admin', 'feedbacks', $project_id ) ) {
```

Comment:
Same as above, $project_id is not yet defined.

Reply:
fixed